x-pack/plugins/maps/public/components/map/feature_tooltip.js

```diff
@@ -98,36 +105,35 @@ export class FeatureTooltip extends React.Component {
           })}
           className="mapFeatureTooltipFilterButton"
         />
-      </EuiFlexItem>
+      </div>
     );
   }
 
   _renderProperties(hasFilters) {
-    return this.state.properties.map((tooltipProperty, index) => {
+    return this.state.properties.map(tooltipProperty => {
       /*
        * Justification for dangerouslySetInnerHTML:
        * Property value contains value generated by Field formatter
        * Since these formatters produce raw HTML, this component needs to be able to render them as-is, relying
        * on the field formatter to only produce safe HTML.
        */
-      const htmlValue = (<span
-        // eslint-disable-next-line react/no-danger
-        dangerouslySetInnerHTML={{
-          __html: tooltipProperty.getHtmlDisplayValue()
-        }}
-      />);
-
-      return (
-        <EuiFlexGroup key={index}>
-          <EuiFlexItem grow={4}>
-            <strong>{tooltipProperty.getPropertyName()}</strong>
-          </EuiFlexItem>
-          <EuiFlexItem grow={6}>
-            {htmlValue}
-          </EuiFlexItem>
+      const htmlValue = (
+        <div>
+          <div
+            className="tooltipLineDescription"
```

Comment:
It's also best to use different class names for different children so if, down the line, you need to add another style to just one of them, you don't need to then create a new class.

```suggestion
            className="mapFeatureTooltip__lineDescription"
```
In the css, you can then just list both class names as the selector like:

```scss
.mapFeatureTooltip__filterButton,
.mapFeatureTooltip__lineDescription {
 ...
}
```